client/Components/Reservation.css

```diff
@@ -1,19 +1,18 @@
-html {
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+*, *:before, *:after {
+  box-sizing: border-box;
+  font-family: Circular, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
 }
 
-/* html {
-  display: grid;
-  grid-template-columns: 2fr 1fr;
-  grid-template-rows: repeat(5, 1fr);
-} */
+.positioner {
```

Comment:
Empty style. Do you still need this?

Reply:
I don't yet but I expect I'll need it to today to spruce up positioning.